### DISCUSSION TITLE
Reflex start abort

### DISCUSSION THREAD

**Discussion by planti** [reflex-dev/reflex#3466] [Q&A]
Hello everyone,
I am developing a simple demo of a RAG application with an LLM and a vector store: LLM on Ollama in the host machine and the vector store in a separate container. I am using the reflex-chat component.
I develop with Visual studio code with the python environment in a container using Visual Studio container development.
Every now and then, I don't understand why, reflex crashes with a "killed" message.  I include an example of the terminal screen.
Some details:
Mac OS Sonoma 14.5, M2 Pro
reflex==0.5.2
reflex-chat==0.0.1
reflex-hosting-cli==0.1.13
Python 3.12.3

Any ideas?

![Screenshot 2024-06-10 alle 12 09 20](https://github.com/reflex-dev/reflex/assets/956527/a114b080-ba8d-4ea2-b409-7b35bfca8b27)

**Comment by picklelo** [comment]
@planti if you run `reflex run --loglevel debug` is there any additional info?

**Comment by planti** [comment]
Hi pickleo, 
this is the output but I don't understand much more:
vscode ➜ /workspace/app $ reflex run --loglevel debug
─────────────────────────────────────────────────────────────────── System Info ───────────────────────────────────────────────────────────────────
Debug: Config file: '/usr/local/lib/python3.12/site-packages/reflex/config.py'
Debug: Config: app_name='app' loglevel=<LogLevel.INFO: 'info'> frontend_port=3000 frontend_path='' backend_port=8000 
api_url='http://localhost:8000' deploy_url='http://localhost:3000' backend_host='0.0.0.0' db_url='sqlite:///reflex.db' redis_url=None 
telemetry_enabled=True bun_path='/home/vscode/.local/share/reflex/bun/bin/bun' cors_allowed_origins=['*'] tailwind={} timeout=120 
next_compression=True react_strict_mode=True frontend_packages=[] cp_backend_url='https://rxcp-prod-control-plane.fly.dev' 
cp_web_url='https://control-plane.reflex.run' gunicorn_worker_class='uvicorn.workers.UvicornH11Worker' gunicorn_workers=None
Debug: Running command: ['/home/vscode/.local/share/reflex/fnm/node-versions/v18.17.0/installation/bin/node', '-v']
Debug: Running command: ['/home/vscode/.local/share/reflex/fnm/fnm', '--version']
Debug: Running command: ['/home/vscode/.local/share/reflex/bun/bin/bun', '-v']
Debug: [Reflex 0.5.4 with Python 3.12.3 (PATH: /usr/local/bin/python)]
Debug: [Node 18.17.0 (Expected: 18.17.0) (PATH:/home/vscode/.local/share/reflex/fnm/node-versions/v18.17.0/installation/bin/node)]
Debug: [FNM 1.35.1 (Expected: 1.35.1) (PATH: /home/vscode/.local/share/reflex/fnm/fnm)]
Debug: [Bun 1.1.10 (Expected: 1.1.10) (PATH: /home/vscode/.local/share/reflex/bun/bin/bun)]
Debug: [OS Linux Debian GNU/Linux 12 (bookworm)]
Debug: Using package installer at: /home/vscode/.local/share/reflex/bun/bin/bun
Debug: Using package executer at: /home/vscode/.local/share/reflex/fnm/node-versions/v18.17.0/installation/lib/node_modules/npm/bin/npm-cli.js
Debug: Unzip path: /usr/bin/unzip
Info: The frontend will run on port 3001.
Info: The backend will run on port 8001.
Info: Overriding config value frontend_port with env var FRONTEND_PORT=3001
Info: Overriding config value backend_port with env var BACKEND_PORT=8001
─────────────────────────────────────────────────────────────── Starting Reflex App ───────────────────────────────────────────────────────────────
DeprecationWarning: rx.input.input has been deprecated in version 0.5.0 use rx.input without the .input suffix. It will be completely removed in 
0.6.0
DeprecationWarning: rx.input.root has been deprecated in version 0.5.0 use rx.input without the .root suffix. It will be completely removed in 
0.6.0
/usr/local/lib/python3.12/site-packages/huggingface_hub/file_download.py:1132: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
Compiling: ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 17/17 0:00:00
Debug: Running command: ['/home/vscode/.local/share/reflex/bun/bin/bun', 'install']
Debug: Installing base frontend packages
Debug: bun install v1.1.10 (5102a944)
Debug: 
Debug: Checked 555 installs across 553 packages (no changes) [323.00ms]
Debug: Running command: ['/home/vscode/.local/share/reflex/bun/bin/bun', 'add', '-d', 'tailwindcss@3.3.2']
Debug: Installing tailwind
Debug: bun add v1.1.10 (5102a944)
Debug: Saved lockfile
Debug: 
Debug: installed tailwindcss@3.3.2 with binaries:
Debug:  - tailwind
Debug:  - tailwindcss
Debug: 
Debug: [168.00ms] done
Debug: Running command: ['/home/vscode/.local/share/reflex/bun/bin/bun', 'add', 'rehype-raw@6.1.1', 'remark-math@5.1.1', '@radix-ui/themes@^3.0.0',
'rehype-katex@6.0.3', 'react-syntax-highlighter@15.5.0', 'remark-gfm@3.0.1', '@emotion/react@^11.11.1', 'lucide-react@0.359.0', 
'@radix-ui/react-progress@^1.0.3', '@chakra-ui/system@2.5.7', 'react-dropzone@14.2.3', '@chakra-ui/react@2.6.1', 'framer-motion@10.16.4', 
'remark-unwrap-images@4.0.0', '@radix-ui/react-form@^0.0.3', 'sonner@1.4.41', 'react-markdown@8.0.7']
Debug: Installing frontend packages from config and components
Debug: bun add v1.1.10 (5102a944)
Debug: Saved lockfile
Debug: 
Debug: installed rehype-raw@6.1.1
Debug: installed remark-math@5.1.1
Debug: installed @radix-ui/themes@3.0.5
Debug: installed rehype-katex@6.0.3
Debug: installed react-syntax-highlighter@15.5.0
Debug: installed remark-gfm@3.0.1
Debug: installed @emotion/react@11.11.1
Debug: installed lucide-react@0.359.0
Debug: installed @radix-ui/react-progress@1.0.3
Debug: installed @chakra-ui/system@2.5.7
Debug: installed react-dropzone@14.2.3
Debug: installed @chakra-ui/react@2.6.1
Debug: installed framer-motion@10.16.4
Debug: installed remark-unwrap-images@4.0.0
Debug: installed @radix-ui/react-form@0.0.3
Debug: installed sonner@1.4.41
Debug: installed react-markdown@8.0.7
Debug: 
Debug: [156.00ms] done
Debug: Running command: ['/home/vscode/.local/share/reflex/fnm/node-versions/v18.17.0/installation/lib/node_modules/npm/bin/npm-cli.js', 'run', 
'next', 'telemetry', 'disable']
Debug: Running command: ['/home/vscode/.local/share/reflex/fnm/node-versions/v18.17.0/installation/bin/node', '-v']
─────────────────────────────────────────────────────────────────── App Running ───────────────────────────────────────────────────────────────────
Debug: Running command: ['/home/vscode/.local/share/reflex/fnm/node-versions/v18.17.0/installation/lib/node_modules/npm/bin/npm-cli.js', 'run', 
'dev']
Debug: Starting frontend
Info: Overriding config value frontend_port with env var FRONTEND_PORT=3001
Info: Overriding config value backend_port with env var BACKEND_PORT=8001
Debug: 
Debug: > dev
Debug: > next dev
Debug: 
Debug:    ▲ Next.js 14.0.1
Debug:    - Local:        http://localhost:3001
App running at: http://localhost:3001
Debug: 
Debug: warning package.json: No license field
DeprecationWarning: rx.input.input has been deprecated in version 0.5.0 use rx.input without the .input suffix. It will be completely removed in 
0.6.0
DeprecationWarning: rx.input.root has been deprecated in version 0.5.0 use rx.input without the .root suffix. It will be completely removed in 
0.6.0
Debug:  ✓ Ready in 17.2s
Debug:  ○ Compiling / ...
Killed
vscode ➜ /workspace/app $ /usr/local/lib/python3.12/site-packages/huggingface_hub/file_download.py:1132: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(
/usr/local/lib/python3.12/multiprocessing/resource_tracker.py:254: UserWarning: resource_tracker: There appear to be 1 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '

**Reply by picklelo** [reply]
@planti does a blank app work for you? I wonder if it's related to some out of memory / other resource error, or if it's a more general reflex issue.
